app/assets/src/components/BulkUploadImport.jsx

```diff
@@ -527,7 +535,15 @@ class BulkUploadImport extends React.Component {
               <form
                 className="bulkSubmitForm"
                 ref="form"
-                onSubmit={this.handleRemoteUploadSubmit}
+                onSubmit={withAnalytics(
+                  this.handleRemoteUploadSubmit,
+                  "BulkUploadImport_upload_form_submitted",
+                  {
+                    samples: this.state.samples.length,
+                    project_id: this.state.projectId,
```

Comment:
Shouldnt we impose camelCase for arguments too ? Or is this constrained by thee server? 
I see some other messages above that use camelCase.

Reply:
Good catch. I'll make them all consistent with the names in the calling context.